packages/asc-ui/src/components/Menu/Menu.tsx

```diff
@@ -0,0 +1,183 @@
+import React from 'react'
+import ownerDocument from '../../utils/ownerDocument'
+import MenuStyle, { MenuStyleProps } from '../../styles/components/MenuStyle'
+import MenuButton from './MenuButton'
+import MenuList from './MenuList'
+import { KeyboardKeys } from '../../types'
+
+type Props = {
+  position?: MenuStyleProps.Position
```

Comment:
consider dropping the namespace constructions. in this context they are not usefull at all.